docs/cli-client/asset/issue-token.md

```diff
@@ -20,10 +20,10 @@ iriscli asset issue-token [flags]
 | --gateway          | string  | false    |               | The unique moniker of the gateway, required when the source is gateway |
 | --symbol           | string  | true     |               | The symbol of the token, length between 3 and 6, alphanumeric characters, case insensitive |
```

Comment:
`Symbol` length should be between 3 and 8.

Please double check all the related docs to make sure their contents match our spec and implementation, and that English-Chinese versions are consistent.

Reply:
Done, double checked